### ISSUE TITLE
No errors are displayed if lighthouse jobs are not created

### ISSUE THREAD

**Issue by ankitm123** [jenkins-x/lighthouse#1242]
It would be nice to post a github comment from lighthouse, if jobs are not created. Atm, I have to look at the lighthouse pod logs to figure out the issue.

**Comment by jstrachan** [comment]
we usually do this on Pull Requests but maybe there are some errors not making it to the comment?

**Comment by ankitm123** [comment]
> we usually do this on Pull Requests but maybe there are some errors not making it to the comment?

yeah the validation type errors. Like for example, name a step with spaces, that normally wont show in the PR, have to see if I can reproduce it in the latest version
